docker/Dockerfile-fargo-master

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.6
```

Comment:
- [ ] That's a conservative version at this point. Would you consider using version 1.7?